2025-03-18 00:48:53,242 - PythonTestLogger - ERROR - Test failed for QF1_P9_P10.py on QF1

SRNs: P9/P10

Test Type: stdin

Test Input:
2 -1 -2      
-4 6 3
-4 -2 8

Expected Output:
P =
0.0 0.0 1.0
1.0 0.0 0.0
0.0 1.0 0.0

L =
1.0 0.0 0.0
1.0 1.0 0.0
-0.5 -0.2 1.0

U =
-4.0 6.0 3.0
0.0 -8.0 5.0
0.0 0.0 0.8


Actual Output:
P =
0.0 0.0 1.0
1.0 0.0 0.0
0.0 1.0 0.0

L =
1.0 0.0 0.0
1.0 1.0 0.0
-0.5 -0.2 1.0

U =
-4.0 6.0 3.0
0.0 -8.0 5.0
0.0 0.0 0.8
--------------------------------------------------
2025-03-18 00:48:56,310 - PythonTestLogger - ERROR - Test failed for QF2_P11_P12.py on QF2

SRNs: P11/P12

Test Type: stdin

Test Input:
1 0 0
0 1 0
0 0 1

Expected Output:
True


Actual Output:
True
--------------------------------------------------
2025-03-18 00:48:58,006 - PythonTestLogger - ERROR - Test failed for QF3_P13_P14.py on QF3

SRNs: P13/P14

Test Type: stdin

Test Input:
1 2 3
4 5 6
7 8 9

Expected Output:
Basis vectors for the column space:
[1.0, 2.0]
[4.0, 5.0]
[7.0, 8.0]
Dimension of the column space: 2


Actual Output:
Basis vectors for the column space:
[1.0, 2.0]
[4.0, 5.0]
[7.0, 8.0]
Dimension of the column space: 2
--------------------------------------------------
2025-03-18 00:48:59,539 - PythonTestLogger - ERROR - Test failed for QA5_P9_P10.py on QA5

SRNs: P9/P10

Test Type: stdin

Test Input:
2 2
3 1
4 2
9 10

Expected Output:
x =
4.0000 -3.0000
Rank = 2
Nullity = 0


Actual Output:
x =
4.0000 -3.0000

Rank = 2
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:00,520 - PythonTestLogger - ERROR - Test failed for QA5_P9_P10.py on QA5

SRNs: P9/P10

Test Type: stdin

Test Input:
3 3
2 -1 3
1 3 2
4 5 -2
5 6 7


Expected Output:
x =
1.2540 0.8413 1.1111
Rank = 3
Nullity = 0

Actual Output:
x =
1.2540 0.8413 1.1111

Rank = 3
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:01,475 - PythonTestLogger - ERROR - Test failed for QA5_P9_P10.py on QA5

SRNs: P9/P10

Test Type: stdin

Test Input:
4 4
4 -2 1 3
1 1 2 1
3 -1 4 2
2 3 -2 5
20 10 30 15

Expected Output:
x =
-10.8065 -6.9355 6.7742 14.1935
Rank = 4
Nullity = 0

Actual Output:
x =
-10.8065 -6.9355 6.7742 14.1935

Rank = 4
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:01,792 - PythonTestLogger - ERROR - Test failed for QA6_P11_P12.py on QA6

SRNs: P11/P12

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA6_P11_P12.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
2 2
1 2
0 1
2 1
2
3

Expected Output:
P' =
8.0
3.0

RREF =
1.0 0.0
0.0 1.0

T^-1 =
1.000 -2.000
0.000 1.000
--------------------------------------------------
2025-03-18 00:49:02,106 - PythonTestLogger - ERROR - Test failed for QA6_P11_P12.py on QA6

SRNs: P11/P12

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA6_P11_P12.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 3
3 1 2
4 2 1
1 2 3
3 1
1
0
1


Expected Output:
P' =
5.0
5.0
4.0

RREF =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0

T^-1 =
0.308 0.077 -0.231
-0.846 0.538 0.385
0.462 -0.385 0.154
--------------------------------------------------
2025-03-18 00:49:02,431 - PythonTestLogger - ERROR - Test failed for QA6_P11_P12.py on QA6

SRNs: P11/P12

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA6_P11_P12.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
4 4
2 3 1 4
1 5 3 2
4 2 6 5
3 1 2 3
4 1
5
6
7
8

Expected Output:
P' =
67.0
72.0
114.0
59.0

RREF =
1.0 0.0 0.0 0.0
0.0 1.0 0.0 0.0
0.0 0.0 1.0 0.0
0.0 0.0 0.0 1.0

T^-1 =
-0.407 0.185 -0.370 1.037
-0.037 0.259 -0.185 0.185
-0.148 0.037 0.259 -0.259
0.519 -0.296 0.259 -0.593

--------------------------------------------------
2025-03-18 00:49:02,706 - PythonTestLogger - ERROR - Test failed for QA7_P13_P14.py on QA7

SRNs: P13/P14

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA7_P13_P14.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 4
1 -1 0 0
0 1 -1 0
0 0 1 -1

Expected Output:
RREF =
1.0 0.0 0.0 -1.0
0.0 1.0 0.0 -1.0
0.0 0.0 1.0 -1.0

Rank = 3
Nullity = 1
--------------------------------------------------
2025-03-18 00:49:02,977 - PythonTestLogger - ERROR - Test failed for QA7_P13_P14.py on QA7

SRNs: P13/P14

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA7_P13_P14.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 3
1 2 3
4 5 6
7 8 10

Expected Output:
RREF =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0

Rank = 3
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:03,239 - PythonTestLogger - ERROR - Test failed for QA7_P13_P14.py on QA7

SRNs: P13/P14

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA7_P13_P14.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
4 5
1 0 -1 2 3
0 1 1 0 -1
1 1 0 -1 4
2 -1 3 1 0

Expected Output:
RREF =
1.0 0.0 0.0 0.0 2.8
0.0 1.0 0.0 0.0 0.5
0.0 0.0 1.0 0.0 -1.5
0.0 0.0 0.0 1.0 -0.7

Rank = 4
Nullity = 1

--------------------------------------------------
2025-03-18 00:49:03,539 - PythonTestLogger - ERROR - Test failed for QA8_P15_P16.py on QA8

SRNs: P15/P16

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA8_P15_P16.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
2 2
2 1
3 4
2 2
5 6
7 8

Expected Output:
C =
2.6 3.2
-0.2 -0.4

RREF =
1.0 0.0
0.0 1.0

Rank = 2
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:03,840 - PythonTestLogger - ERROR - Test failed for QA8_P15_P16.py on QA8

SRNs: P15/P16

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA8_P15_P16.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 3
3 1 2
4 2 1
1 2 3
3 2
6 9
3 7
8 5


Expected Output:
C =
0.2 2.2
-0.4 -1.9
2.8 2.2

RREF =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0

Rank = 3
Nullity = 0
--------------------------------------------------
2025-03-18 00:49:04,174 - PythonTestLogger - ERROR - Test failed for QA8_P15_P16.py on QA8

SRNs: P15/P16

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QA8_P15_P16.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
4 4
2 3 1 4
1 5 3 2
4 2 6 5
3 1 2 3
4 2
5 6
7 8
9 10
11 12

Expected Output:
C =
7.3 7.8
2.0 2.2
-1.0 -1.1
-3.7 -3.8

RREF =
1.0 0.0 0.0 0.0
0.0 1.0 0.0 0.0
0.0 0.0 1.0 0.0
0.0 0.0 0.0 1.0

Rank = 4
Nullity = 0

--------------------------------------------------
2025-03-18 00:49:04,436 - PythonTestLogger - ERROR - Test failed for QF4_P15_P16.py on QF4

SRNs: P15/P16

Test Type: stdin

Test Input:
2
2 -1
1 3

Expected Output:
U =
1.0 0.0
0.0 1.0

C =
0.4 0.1
-0.1 0.3

A^-1 =
0.4 0.1
-0.1 0.3


Actual Output:
U =
1.0 0.0
0.0 1.0

C =
0.4 0.1
-0.1 0.3

A^-1 =
0.4 0.1
-0.1 0.3
--------------------------------------------------
2025-03-18 00:49:04,714 - PythonTestLogger - ERROR - Test failed for QF4_P15_P16.py on QF4

SRNs: P15/P16

Test Type: stdin

Test Input:
3
3 1 2
4 2 1
1 2 3

Expected Output:
U =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0

C =
0.3 0.1 -0.2
-0.8 0.5 0.4
0.5 -0.4 0.2

A^-1 =
0.3 0.1 -0.2
-0.8 0.5 0.4
0.5 -0.4 0.2


Actual Output:
U =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0

C =
0.3 0.1 -0.2
-0.8 0.5 0.4
0.5 -0.4 0.2

A^-1 =
0.3 0.1 -0.2
-0.8 0.5 0.4
0.5 -0.4 0.2
--------------------------------------------------
2025-03-18 00:49:05,293 - PythonTestLogger - ERROR - Test failed for QF5_P17_P18.py on QF5

SRNs: P17/P18

Test Type: stdin

Test Input:
2
2 -1 1
1 3 2

Expected Output:
U =
1.0 -0.5 
0.0 1.0 

x =
0.7 0.4

Actual Output:
U =
1.0 -0.5
0.0 1.0

x =
0.7 0.4
--------------------------------------------------
2025-03-18 00:49:06,179 - PythonTestLogger - ERROR - Test failed for QF6_P19_P20.py on QF6

SRNs: P19/P20

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QF6_P19_P20.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 3
1 2 -1
2 4 -2
3 6 -3

Expected Output:
RREF =
1.0 2.0 -1.0
0.0 0.0 0.0
0.0 0.0 0.0
--------------------------------------------------
2025-03-18 00:49:06,458 - PythonTestLogger - ERROR - Test failed for QF6_P19_P20.py on QF6

SRNs: P19/P20

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QF6_P19_P20.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 3
1 2 3
0 1 4
5 6 0

Expected Output:
RREF =
1.0 0.0 0.0
0.0 1.0 0.0
0.0 0.0 1.0
--------------------------------------------------
2025-03-18 00:49:06,748 - PythonTestLogger - ERROR - Test failed for QF6_P19_P20.py on QF6

SRNs: P19/P20

Error:
Traceback (most recent call last):
  File "/Volumes/Aditya SSD/Project/AutomaticPythonTesting/AutomaticPythonTesting/scripts/QF6_P19_P20.py", line 3, in <module>
    from sympy import Matrix
ModuleNotFoundError: No module named 'sympy'


Test Type: stdin

Test Input:
3 4
4 7 2 5
2 6 5 8
1 3 1 4

Expected Output:
RREF =
1.0 0.0 0.0 -2.6
0.0 1.0 0.0 2.2
0.0 0.0 1.0 0.0
--------------------------------------------------
2025-03-18 00:49:07,764 - PythonTestLogger - ERROR - Test failed for QA1_P1_P2.py on QA1

SRNs: P1/P2

Test Type: stdin

Test Input:
3
2 -1 -2
-4 6 3
-4 -2 8
3 7 10

Expected Output:
L =
[[ 1.    0.    0.  ]
 [ 1.    1.    0.  ]
 [-0.5  -0.25  1.  ]]

U =
[[-4.    6.    3.  ]
 [ 0.   -8.    5.  ]
 [ 0.    0.    0.75]]

x =
26.62 9.92 16.67

Actual Output:
CHECK [[ 2. -1. -2.]
 [-4.  6.  3.]
 [-4. -2.  8.]]
L =
[[ 1.    0.    0.  ]
 [ 1.    1.    0.  ]
 [-0.5  -0.25  1.  ]]

U =
[[-4.    6.    3.  ]
 [ 0.   -8.    5.  ]
 [ 0.    0.    0.75]]

x =
26.62 9.92 16.67
--------------------------------------------------
2025-03-18 00:49:08,711 - PythonTestLogger - ERROR - Test failed for QA1_P1_P2.py on QA1

SRNs: P1/P2

Test Type: stdin

Test Input:
4
4  1  0  2
1  5  3 -1
2  0  6  1
3  2  1  7
10 8 15 20

Expected Output:
L =
[[ 1.    0.    0.    0.  ]
 [ 0.25  1.    0.    0.  ]
 [ 0.5  -0.11  1.    0.  ]
 [ 0.75  0.26  0.03  1.  ]]

U =
[[ 4.    1.    0.    2.  ]
 [ 0.    4.75  3.   -1.5 ]
 [ 0.    0.    6.32 -0.16]
 [ 0.    0.    0.    5.9 ]]

x =
1.43 0.64 1.72 1.81

Actual Output:
CHECK [[ 4.  1.  0.  2.]
 [ 1.  5.  3. -1.]
 [ 2.  0.  6.  1.]
 [ 3.  2.  1.  7.]]
L =
[[ 1.    0.    0.    0.  ]
 [ 0.25  1.    0.    0.  ]
 [ 0.5  -0.11  1.    0.  ]
 [ 0.75  0.26  0.03  1.  ]]

U =
[[ 4.    1.    0.    2.  ]
 [ 0.    4.75  3.   -1.5 ]
 [ 0.    0.    6.32 -0.16]
 [ 0.    0.    0.    5.9 ]]

x =
1.43 0.64 1.72 1.81
--------------------------------------------------
2025-03-18 00:49:09,662 - PythonTestLogger - ERROR - Test failed for QA1_P1_P2.py on QA1

SRNs: P1/P2

Test Type: stdin

Test Input:
5
5  2  1  0  3
2  6  0  1 -1
1  0  7  2  4
0  1  2  8  0
3 -1  4  0  9
12 8 25 18 30

Expected Output:
L =
[[ 1.    0.    0.    0.    0.  ]
 [ 0.4   1.    0.    0.    0.  ]
 [ 0.2  -0.08  1.    0.    0.  ]
 [ 0.    0.19  0.31  1.    0.  ]
 [ 0.6  -0.42  0.48 -0.08  1.  ]]

U =
[[ 5.    2.    1.    0.    3.  ]
 [ 0.    5.2  -0.4   1.   -2.2 ]
 [ 0.    0.    6.77  2.08  3.23]
 [ 0.    0.    0.    7.17 -0.57]
 [ 0.    0.    0.    0.    4.68]]

x =
-0.36 1.67 1.42 1.69 3.01

Actual Output:
CHECK [[ 5.  2.  1.  0.  3.]
 [ 2.  6.  0.  1. -1.]
 [ 1.  0.  7.  2.  4.]
 [ 0.  1.  2.  8.  0.]
 [ 3. -1.  4.  0.  9.]]
L =
[[ 1.    0.    0.    0.    0.  ]
 [ 0.4   1.    0.    0.    0.  ]
 [ 0.2  -0.08  1.    0.    0.  ]
 [ 0.    0.19  0.31  1.    0.  ]
 [ 0.6  -0.42  0.48 -0.08  1.  ]]

U =
[[ 5.    2.    1.    0.    3.  ]
 [ 0.    5.2  -0.4   1.   -2.2 ]
 [ 0.    0.    6.77  2.08  3.23]
 [ 0.    0.    0.    7.17 -0.57]
 [ 0.    0.    0.    0.    4.68]]

x =
-0.36 1.67 1.42 1.69 3.01
--------------------------------------------------
